### PR TITLE
BCDA-4292: Added language for the DELETE endpoint and a side navigation item

### PR DESCRIPTION
**Pull request by Dave-Sukharan**
Added Delete Endpoint Language to the text including cURL commands and explanation. Added the side navigation item to skip to the Delete section.

### Fixes [BCDA-4292](https://jira.cms.gov/browse/BCDA-4292)

BCDA created a new endpoint allowing users to end active jobs. This update adds documentation to the Build page to demonstrate the DELETE endpoint.

### Proposed Changes

- Add DELETE endpoint section
- Add endpoint cURL command
- Add explanation

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [X] no PHI/PII is affected by this change

### Feedback Requested

Is the endpoint language as expected? Are there any missing pieces we should inform users of?

### CODE REVIEW

File: _includes/build/deleting_jobs.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<h3>Step 1: Obtain an Access Token</h3>
+	<p>
+		See <a href="#authorization" class="in-text__link">Authentication and Authorization</a> above.
+	</p>
+<h3>Step 2: Send a Request to Delete a Job</h3>
+
+<h4>Request to Delete an Active Job</h4>
+      <pre><code>DELETE /api/v1/jobs/{jobId}</code></pre>
+
+<h4>Request Headers</h4>
```
Comment (mtrang1263) [inline _includes/build/deleting_jobs.html:10]: We actually do not require any headers to be present

File: _includes/build/deleting_jobs.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<h3>Step 1: Obtain an Access Token</h3>
```
Comment (mtrang1263) [inline _includes/build/deleting_jobs.html:1]: Can we change this language/page to say "Cancel a Job"?

File: _includes/build/cancelling_jobs.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<h3>Step 1: Obtain an Access Token</h3>
+	<p>
+		See <a href="#authorization" class="in-text__link">Authentication and Authorization</a> above.
+	</p>
+<h3>Step 2: Send a Request to Delete a Job</h3>
```
Comment (mtrang1263) [inline _includes/build/cancelling_jobs.html:5]: Can we change Delete to Cancel?

Comment (Dave-Sukharan) [reply _includes/build/cancelling_jobs.html]: Done! Corrected a few typos that were in there as well

**Review by mtrang1263 — APPROVED** [review]

**Pull request merged**